src/routes/userRoutes.js

```diff
@@ -0,0 +1,69 @@
+const { Router } = require("express");
+const express = require("express");
+const { body, validationResult } = require("express-validator");
+const User = require("../models/User");
+const bcrypt = require("bcryptjs");
+
+const router = Router();
+
+router.use(express.json());
+//new user registration
+router.post(
+  "/user_create",
+  body("firstName")
+    .isAlpha("en-US", { ignore: " " })
+    .withMessage("First name can only contain letters"),
+  body("lastName")
+    .isAlpha("en-US", { ignore: " " })
+    .withMessage("Last name can only contain letters"),
+  body("email").isEmail().withMessage("Invalid email address"),
+  body("password")
+    .isLength({ min: 5 })
+    .withMessage("Password must be at least 5 chars long"),
+  async (req, res) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+      return res
+        .status(400)
+        .json({ msg: "form validation failed", err: errors.array() });
```

Comment:
it's better to add a json format by default with `app.use` in a server initialization part

Reply:
it's better to have the function called `formatResponse`, which is used whenever you send the response to client. 
it will help us to keep the same format and change everything fast if there is a need for that.

```
formatResponse = (data = null, err = null, msg = null) => ({
  data,
  err,
  msg
})
```

Reply:
> it's better to add a json format by default with app.use in a server initialization part

I added app.use(express.json()) into index.js.

> it's better to have the function called formatResponse, which is used whenever you send the response to client.
> it will help us keep the same format and change everything fastly if there is a need for that.

I created a separate file with formatResponse function (/src/helpers/serverResponse.js).